armi/bookkeeping/db/passiveDBLoadPlugin.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2024 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Provides the ability to ignore parameters sections of blueprint files.
+
+This plugin can allow you to more easily open a database, because you can ignore sections of the
+blueprint files, and ignore any parameters as you want.
+
+This was designed to allow loading an ARMI database without the application that created it.
+"""
+import yamlize
+
+from armi import plugins
+from armi.reactor import parameters
+from armi.reactor.parameters import ParamLocation
+from armi.utils import units
+
+
+class PassThroughYamlize(yamlize.Object):
+    """Just a helper for PassiveDBLoadPlugin, to allow for ignore unknown blueprints sections."""
+
+    @classmethod
+    def from_yaml(cls, loader, node, round_trip_data=None):
+        return yamlize.Object.from_yaml.__func__(
+            PassThroughYamlize, loader, node, round_trip_data
+        )
+
+
+# TODO: JOHN needs testing!
+class PassiveDBLoadPlugin(plugins.ArmiPlugin):
+    """Provides the ability to ignore parameters sections of blueprint files.
+
+    This plugin allows you to more easily open a database, because you can ignore sections of the
+    blueprint files and as many of the parameters as you want.
```

Comment:
Just to clarify, this isn't skipping the parameters right? You can still look at that parameter's value, you just don't necessarily know what it means or what its units are without outside knowledge.

Reply:
The parameters are still loading, but with defaulted / dummy definitions and units. And they won't be attached to any range validation.